lib/reporting/protocols_report.rb

```diff
@@ -210,6 +215,48 @@ def saml_signature_issues_table
       ]
     end
 
+    def loa_acr_requests_table
+      [
+        ['Count of integrations using LOA', 'List of issuers with the issue'],
+        [
+          loa_issuers_data.length,
+          loa_issuers_data.join(', '),
+        ],
+      ]
+    end
+
+    def loa_issuers_data
+      @loa_issuers_data ||= begin
+        cloudwatch_client.fetch(
+          query: loa_issuers_query,
+          from: time_range.begin,
+          to: time_range.end,
+        ).
+          map { |slice| slice['issuer'] }.
+          uniq
```

Comment:
[question] since you're running dedup in the query, is this necessary?

Reply:
It is needed because the same issuer may be present in different time slices; the `dedup` would only make them unique within a slice.